.github/workflows/publish-js-sdks.yml

```diff
@@ -17,20 +17,26 @@ jobs:
           node-version-file: "package.json"
           registry-url: "https://registry.npmjs.org"
 
+      - uses: pnpm/action-setup@v4
+        name: Install pnpm
+        with:
+          run_install: true
+
       - name: Install dependencies
-        run: npm install
-        working-directory: sdk/js/
+        run: pnpm --filter @pythnetwork/express-relay-js install
 
       - name: Run tests
-        run: npm test # Ensure your tests pass before publishing
-        working-directory: sdk/js/
+        run: pnpm --filter @pythnetwork/express-relay-js test # Ensure your tests pass before publishing
```

Comment:
I'd get rid of the `--filter` and use `-r` instead; that way pnpm will just run `test` on any package it finds in the workspace which seems more sementically intentional